tests/net/utils/src/main.c

```diff
@@ -821,7 +821,7 @@ ZTEST(test_utils_fn, test_addr_parse)
 			zassert_true(
 				net_ipv4_addr_cmp(
 				      &net_sin(&addr)->sin_addr,
-				      &parse_ipv4_entries[i].result.sin_addr),
+				      &parse_ipv4_entries[i].result.sin_addr) ==
 				parse_ipv4_entries[i].address);
```

Comment:
This check needs to be
```suggestion
                                    &parse_ipv4_entries[i].result.sin_addr) ==
                               parse_ipv4_entries[i].verdict);
```